README.md

```diff
@@ -61,6 +61,39 @@ module "vpc" {
 For Terraform version older than 0.7.0 use `ref=v1.0.0`:
 `source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.0"`
 
+Usage with other modules
+------------------------
+
+There are situations where other modules will be used alongside this module, but
+all the networking components out of this module (such as route tables, NAT
+Gateways and Internet Gateways) needs to be provisioned first before resources
+(such as EC2 instances) can be created in other modules. Use of `depends_id` as
+an output variable provides a suitable workaround.
+
+Here is an example of usage alongside the `tf_aws_bastion_s3_keys` community
+module so that the bastion host is only created once all resources in the VPC
+module has been created.
+
+```hcl
+module "vpc" {
+  source = "github.com/terraform-community-modules/tf_aws_vpc"
+  ...
+}
+
+module "bastion" {
+  source                      = "github.com/terraform-community-modules/tf_aws_bastion_s3_keys"
+  instance_type               = "t2.micro"
+  ami                         = "ami-123456"
+  region                      = "eu-west-1"
+  iam_instance_profile        = "s3_readonly"
+  s3_bucket_name              = "public-keys-demo-bucket"
+  vpc_id                      = "vpc-123456"
+  subnet_ids                  = ["subnet-123456", "subnet-6789123", "subnet-321321"]
```

Comment:
These subnets will not be hardcoded here, but rather used from outputs from the `${module.vpc.public_subnet_ids}` (for example). Doing so Terraform will be able to resolve dependencies and make resources in the correct order.

Even more, if the creation of some VPC resources will take some time (lets say related to `elasticcache`) I don't want the creation of bastion host to be delayed (or even dependent on it).